### PR TITLE
[zh-CN]: Remove outdated content for compatibility.

### PR DESCRIPTION
**Pull request by loveholly**
Sync from the en content, and you can checkout it out here: https://github.com/mdn/content/commit/e452bf7276167b8b9dae27df697ca4af0f2c5177#diff-ab9b03936ee189615c9bc44ec0f23a0c5f1986a14342a91eca55b1ef504ae3f2L20 for more details.

### CODE REVIEW

File: files/zh-cn/web/javascript/guide/modules/index.md [hunk 1]
```diff
@@ -13,11 +13,7 @@ JavaScript 程序本来很小——在早期，它们大多被用来执行独立
 
 因此，近年来，有必要开始考虑提供一种**将 JavaScript 程序拆分为可按需导入的单独模块**的机制。Node.js 已经提供这个能力很长时间了，还有很多的 JavaScript 库和框架已经开始了模块的使用（例如，[CommonJS](https://en.wikipedia.org/wiki/CommonJS) 和基于 [AMD](https://github.com/amdjs/amdjs-api/blob/master/AMD.md) 的其他模块系统 如 [RequireJS](https://requirejs.org/)，以及最新的 [Webpack](https://webpack.github.io/) 和 [Babel](https://babeljs.io/)）。
 
-好消息是，最新的浏览器开始原生支持模块功能了，这是本文要重点讲述的。这会是一个好事情 —- 浏览器能够最优化加载模块，使它比使用库更有效率：使用库通常需要做额外的客户端处理。
-
-## 浏览器兼容性
-
-{{Compat}}
+目前现代流浏览器都已经不需要额外转译就原生支持了模块化功能。这是件好事——浏览器可以优化模块加载，这比使用一个单独的库进行额外的客户端处理和额外的网络开销更高效。不过，这并不会使像 Webpack 这类的打包工具过时——打包工具仍然在将代码划分为合理大小的块方面表现出色，并且能够进行其他优化，如代码压缩、消除无用代码和树摇优化。
```
Comment (yin1999) [inline files/zh-cn/web/javascript/guide/modules/index.md:16]:
```suggestion
目前现代流浏览器都已经不需要额外转译就原生支持了模块化特性。这是件好事——浏览器可以优化模块加载，这比使用一个单独的库进行额外的客户端处理和额外的网络开销更高效。不过，这并不会使像 Webpack 这类的打包工具过时——打包工具仍然在将代码划分为合理大小的分块方面表现出色，并且能够进行其他优化，如极简化、消除无用代码和摇树优化。
```

File: files/zh-cn/web/javascript/guide/modules/index.md [hunk 2]
```diff
@@ -13,11 +13,7 @@ JavaScript 程序本来很小——在早期，它们大多被用来执行独立
 
 因此，近年来，有必要开始考虑提供一种**将 JavaScript 程序拆分为可按需导入的单独模块**的机制。Node.js 已经提供这个能力很长时间了，还有很多的 JavaScript 库和框架已经开始了模块的使用（例如，[CommonJS](https://en.wikipedia.org/wiki/CommonJS) 和基于 [AMD](https://github.com/amdjs/amdjs-api/blob/master/AMD.md) 的其他模块系统 如 [RequireJS](https://requirejs.org/)，以及最新的 [Webpack](https://webpack.github.io/) 和 [Babel](https://babeljs.io/)）。
```
Comment (yin1999) [inline files/zh-cn/web/javascript/guide/modules/index.md:14]:
```suggestion
复杂的项目需要一种**将 JavaScript 程序拆分为可按需导入的单独模块**的机制。Node.js 已经提供这个能力很长时间了，还有很多的 JavaScript 库和框架已经开始了模块的使用（例如，[CommonJS](https://zh.wikipedia.org/wiki/CommonJS) 和基于 [AMD](https://github.com/amdjs/amdjs-api/blob/master/AMD.md) 的其他模块系统 如 [RequireJS](https://requirejs.org/)、[Webpack](https://webpack.js.org/) 和 [Babel](https://babeljs.io/)）。
```

**Review by yin1999 — APPROVED** [review]
Thank you @loveholly

**Comment by loveholly** [comment]
> Thank you @loveholly

My pleasure.

**Pull request merged**